toolium/selenoid.py

```diff
@@ -0,0 +1,199 @@
+# -*- coding: utf-8 -*-
+u"""
+Copyright 2019 Telefónica Investigación y Desarrollo, S.A.U.
+This file is part of Toolium.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import os
+import time
+import requests
+
+from toolium.format_utils import get_valid_filename
+
+# constants
+STATUS_OK = 200
+DOWNLOADS_PATH = u'downloads'
+MP4_EXTENSION = u'mp4'
+LOG_EXTENSION = u'log'
+
+
+class Selenoid(object):
+    """
+    toolium.cfg or local.toolium.cfg properties files:
```

Comment:
default properties file names in toolium are properties.cfg and local-properties.cfg

Reply:
Fixed in 2bcebd5